packages/plans-grid-next/src/components/features-grid/plan-features-container.tsx

```diff
@@ -1,10 +1,10 @@
 import { JetpackLogo } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import { useManageTooltipToggle } from '../hooks/use-manage-tooltip-toggle';
-import { DataResponse, GridPlan } from '../types';
-import PlanFeatures2023GridFeatures from './features';
-import PlanDivOrTdContainer from './plan-div-td-container';
-import { Plans2023Tooltip } from './plans-2023-tooltip';
+import { useManageTooltipToggle } from '../../hooks/use-manage-tooltip-toggle';
+import { DataResponse, GridPlan } from '../../types';
+import PlanDivOrTdContainer from '../shared/plan-div-td-container';
```

Comment:
Is this used by anything other than the Features Grid? 🤔

Reply:
Ah, yes I think this can be moved into `/features-grid/**` 👍